checkstyle.properties

```diff
@@ -1,20 +1,26 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18
-publicVersions=4.19
+archivedVersions=2.2,2.3,2.4,3.5.1,3.6,3.6.1,3.7,4.7,4.8,4.9,4.10,4.11,4.16,4.17,4.18,4.19
+publicVersions=4.20
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+4.20.description=Upgrade to Checkstyle 8.20
+4.20.sqVersions=[6.7,LATEST]
+4.20.date=2019-05-14
+4.20.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.20
+4.20.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/4.20/checkstyle-sonar-plugin-4.20.jar
+
 4.19.description=Upgrade to Checkstyle 8.19
 4.19.sqVersions=[6.7,LATEST]
 4.19.date=2019-04-29
 4.19.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/4.19
 4.19.downloadUrl=https://github.com/checkstyle/sonar-checkstyle/releases/download/4.19/checkstyle-sonar-plugin-4.19.jar
 
 4.18.description=Upgrade to Checkstyle 8.18
-4.18.sqVersions=[6.7,7.6+]
+4.18.sqVersions=[6.7,7.7+]
```

Comment:
No point in this update. If there were, you'd use a `*` instead of a `+`. But archived compatibility is used only for helping users upgrade _from_ these archived versions. By practice and convention this ending range would stay set to the SQ version that was current when this plugin version was archived.

Reply:
ok, good to know! i have reverted this change